Include/internal/pycore_code.h

```diff
@@ -24,9 +25,103 @@ struct _PyOpcache {
     char optimized;
 };
 
+
+// We would use an enum if C let us specify the storage type.
+typedef unsigned char _PyFastLocalKind;
+// Note that these all fit within _PyFastLocalKind, as do combinations.
+#define CO_FAST_POSONLY     0x01
+#define CO_FAST_POSORKW     0x02
+#define CO_FAST_VARARGS     0x04
+#define CO_FAST_KWONLY      0x08
+#define CO_FAST_VARKWARGS   0x10
+#define CO_FAST_LOCALONLY   0x20
+#define CO_FAST_CELL        0x40
+#define CO_FAST_FREE        0x80
```

Comment:
Likewise for these two.

Reply:
Likewise `kind & (CO_FAST_CELL | CO_FAST_FREE)` is simpler than `kind == CO_FAST_CELL || kind == CO_FAST_FREE`.

Reply:
`kind & (CO_FAST_CELL | CO_FAST_FREE)` should still work fine.

Reply:
Huh? If the flag bits were given consecutive values, that would produce true if the kind was one of the other values that shares a bit with these constants.

Reply:
While they are mutually exclusive, I'd rather leave them alone.  I tried changing it but it didn't make anything clearer while making the code messier.

Reply:
Let's leave them alone. We have plenty of bits.